playbooks/group_vars/all.yml

```diff
@@ -14,3 +14,11 @@
 
 # This is the location where the rpc repository was checked out
 rpc_repo_path: /opt/os-ansible-deployment
+
+# This is the OSAD version you are deploying rpc-extras against
+rpc_release: 11.0.1
```

Comment:
@d34dh0r53 @nrb @stevelle @prometheanfire With extras being the focus of the install experience, with it's own release timing/version numbers, I wonder about this line.

We're using a submodule to point to a specific version of OSAD I thought?. So, is the description just wrong? In fact, whatever you put in that value has no effect on what is actually deployed from OSAD, and is only used for release version files and such?